### PR TITLE
Moved interactive shell to "manage" to allow more commands

### PR DESCRIPTION
**Pull request by rochacbruno**
- Moved from manage.py to **manage** package
- Renamed module config.settings to config.base due to naming conflict (there already an object named config.settings)
- Added manage>=0.1.10 as optional requirement
- Deleted **manage.py** and added **manage.yml** file specifying interactive shell auto imported objects and custom commands

Robottelo interactive shell `$python manage.py shell` was broken, some **rt** objects having problems to be imported. So I moved to [manage library](http://github.com/rochacbruno/manage)

The plan is to add more subcommands such as `manage ui browse content_host.search` and then a new browsing session will be opened in configured driver and an interactive shell will be opened allowing the execution of **ui** commands e.g: `rt.ui.contenthost.click(locators.something)`

This PR is the first step, now the new commands will be created and included in `manage.yml` file and now to open the interactive shell:

```console
$ cd robottelo
$ pip install -r requirements-optional.txt

#or
$ pip install manage
$ pip install ipython

# and then

$ manage --help
$ manage shell
```

Example: `manage shell`

manage

I'll work in the new command `manage ui browse <feature_name>` in a new PR #3702

### CODE REVIEW

File: manage.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  auto_import:
+    display: true
+    objects:
+        robottelo.config.settings:
+          as: settings
+          init: configure
+        nailgun.entities:
+          as: entities
+        nailgun:
+          as: ng
```
Comment (elyezer) [inline manage.yml]: Using the entire name seems to be better to me.

Same for rt.

File: manage.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      sys.path.insert(0, '.')
+      sys.path.insert(0, 'robottelo')
+      for item in pkgutil.walk_packages(['robottelo']):
+          import_string('robottelo.{0}'.format(item[1]))
+  readline_enabled: true
+
+# commands
+click_commands: []
+function_commands: []
+inline_commands: []
```
Comment (elyezer) [inline manage.yml]: Add a line break at the end?

File: manage.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        nailgun:
+          as: ng
+        robottelo:
+          as: rt
+        robottelo.ui.locators:
+          as: locators
+  banner:
+    enabled: true
+    message: Welcome to {project_name} Interactive shell
+  init_script: |
```
Comment (elyezer) [inline manage.yml]: I am wondering if `manage` can support a flag or something to make this automatically.

I mean if I have something like the following on the config YAML file:

```yaml
shell:
  auto_import:
      objects:
        robottelo:
          as: robottelo
          import_submodules: true
```

That will make `robottelo` and all submodules of robottelo available as `robottelo.<submodule?` available.

Also the `import_submodules` can be a list telling what to import or using `*` as the first item to tell to import all available submodules. The list approach is more promising in my opinion.

**Comment by elyezer** [comment]
ACK, pending comments.

**Comment by rochacbruno** [comment]
@elyezer all comments addressed

- included the full module name (but kept rt and ng for short)
- moved submodules import logic in to manage package

File: manage.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        nailgun.entities:
+          as: entities
+        nailgun:
+          as:
+            - ng
+            - nailgun
+        robottelo:
+          as:
+            - rt
+            - robottelo
```
Comment (sghai) [inline manage.yml]: :+1:

File: manage.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    objects:
+        robottelo.config.settings:
+          as: settings
+          init: configure
+        nailgun.entities:
+          as: entities
+        nailgun:
+          as:
+            - ng
+            - nailgun
```
Comment (sghai) [inline manage.yml]: :+1:

**Comment by sghai** [comment]
All comments addressed and looks good to me. Merging now.

**Pull request merged**